stdlib/public/core/FloatingPointParsing.swift.gyb

```diff
@@ -128,8 +128,10 @@ extension ${Self}: LosslessStringConvertible {
   ///     ${Self}("0x1.25e4")  // Incorrect exponent format
   ///
   /// - Parameter text: The input string to convert to a `${Self}` instance. If
-  ///   `text` has invalid characters or is in an invalid format, the result
-  ///   is `nil`.
+  ///   `text` has invalid characters or is in an invalid format, the result is
+  ///   `nil`.  If the input is valid but the parsed value would be too small
+  ///   or large for the type, the result will be a zero or infinity with the
+  ///   appropriate sign.
```

Comment:
```suggestion
  ///   `nil`.  If the input is valid but the parsed value would be too small
  ///   or large for the type, the result is a zero or infinity with the
  ///   appropriate sign.
```

Using present tense, to match.

Are there two possible values (zero, inf) or three (zero, -inf, +inf)?

Reply:
"a zero or infinity with appropriate sign" is correct; ±0, ±∞.

Reply:
It sounds like the mapping might as follows?

-infinity: Negative number with a magnitude too large to represent
+infinity: Negative number with a magnitude too large to represent
-zero: Negative number with a magnitude too small to represent
+zero: Positive number with a magnitude small to represent

If that's correct, we should say so explicitly.  Also, how should a developer detect the difference between a string literal that was actually "0" and one whose magnitude is too small to represent that produced a +/- zero value?

Reply:
Currently, the API provides no way to distinguish the response for "0" and "1e-50000"; both will return +0.

Reply:
The structure of this whole description is a little awkward.  I'd like to reword it a bit so that it's organized to describe how each kind of input is parsed:  that is, we explain that "inf" and "infinity" (case insensitive) yield a +∞, rather than saying that *infinity* values have a particular representation.

Reply:
I adjusted the wording along these lines.  Let me know if that helps.